### PR TITLE
adding logback + slf4j logging

### PR DESCRIPTION
**Pull request by IvanGurtler**
testing on EAP6.3 (wildfly  problems can be solved by jboss-deployment-structure.xml  ... but not tested on wildfly)

### CODE REVIEW

File: model/api/src/main/java/org/jboss/aerogear/unifiedpush/utils/AeroGearLogger.java [hunk 1]
```diff
@@ -46,23 +46,23 @@ public void info(String message){
     }
 
     public void warning(String message){
-        AeroGearLogger.logger.log(Level.WARNING, format(message));
+        AeroGearLogger.logger.warn(format(message));
```
Comment (matzew) [inline model/api/src/main/java/org/jboss/aerogear/unifiedpush/utils/AeroGearLogger.java]: just wondering, w/ slf4j do we still need the custom `format()` ? I hope/think that's done by the slf4j logging library

File: servers/ups-as7/src/main/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+			<pattern>%date %level [%thread] %logger{10} [%file:%line] %msg%n
+			</pattern>
+		</encoder>
+		<filter class="ch.qos.logback.classic.filter.LevelFilter">
+			<level>ERROR</level>
+			<onMatch>ACCEPT</onMatch>
+			<onMismatch>DENY</onMismatch>
+		</filter>		
+	</appender>
+	<logger name="org.springframework" level="ERROR" />
```
Comment (matzew) [inline servers/ups-as7/src/main/resources/logback.xml]: @IvanGurtler ok if we get rid of the springframework here ? :)

**Comment by matzew** [comment]
@IvanGurtler thanks a lot, will do some tests/tweaks on this next week.

But thanks for contributing !

File: servers/ups-as7/src/main/resources/logback.xml (hunk 2 above)
Comment (IvanGurtler) [inline servers/ups-as7/src/main/resources/logback.xml]: Sorry it was copy and paste :)

File: model/api/src/main/java/org/jboss/aerogear/unifiedpush/utils/AeroGearLogger.java (hunk 1 above)
Comment (IvanGurtler) [inline model/api/src/main/java/org/jboss/aerogear/unifiedpush/utils/AeroGearLogger.java]: yes ... it is true .. I changed it

**Comment by matzew** [comment]
@IvanGurtler I have created this PR, containing your commints + a first draft of my changes.

closing this one, since the other PR contains still you fixes :-) 

Thanks again for sending us the PR

**Pull request closed without merging**